common/src/main/java/io/homeassistant/companion/android/common/sensors/NetworkSensorManager.kt

```diff
@@ -148,7 +161,7 @@ class NetworkSensorManager : SensorManager {
             wifiSignalStrength
         )
         val list = if (hasWifi(context)) {
-            val withPublicIp = wifiSensors.plus(publicIp)
+            val withPublicIp = wifiSensors.plus(publicIp).plus(ip6Addresses)
```

Comment:
this sensor requires the user to have Android 6 or higher so we need to make sure we do not make the sensor available to all user as the app supports Android 5+

Reply:
Yes, I fixed that in a later commit. The change should be in here now